tests/functional/test_notebooks.py

```diff
@@ -59,8 +59,9 @@ class TestNotebooksCreate(object):
         "machineType": "P5000",
         "container": "jupyter/notebook",
         "clusterId": "321",
-        'isPreemptible': False,
-        'isPublic': False,
+        "isPreemptible": False,
+        "isPublic": False,
+        "workspace": "none",
```

Comment:
I wanted to finally drop passing "none" to workspace. The PR to that is waiting for review. Replace this with some URL or remove the line, please.

Reply:
After attempting the change, I realized that we still want this test for `"none"`. This is the expected request json the notebook endpoints depend on sending an explicit 'none' for no workspace and that can't really change until we retire v1. I have updated the all options test to use an explicit url.